config/routes.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  root 'root#index'
+  namespace :api do
+    namespace :v1 do
+      get 'greetings/random', to: 'greetings#random'
+    end
+  end
+end
```

Comment:
- [ ] Kindly fix the following routing error so that every time the user refreshes the page, a random greeting is displayed or every time they click the greeting button.

Error encountering the user
[cinnamon-2023-11-23T235637+0200.webm](https://github.com/mariegrace31/hello-rails-react/assets/81932478/5f4b2d37-7de5-47f3-a4d8-551ea53f5d35)